spec/1.1/index.bs

```diff
@@ -36,9 +36,7 @@ urlPrefix: https://www.w3.org/TR/html5/
 </pre>
 
 
-<b style="color: red; font-size: 1.3em">Deprecated API</b>
-
-<b>The version of the WebVR API represented in this document is does not represent the final shape of the API. While the API is being finalized support for this version may temporarily be available in some browsers but is expected to be replaced eventually.</b>
+<b>Development of WebVR 1.1 specification is finished. The next version of the WebVR specification is being developed under a WebVR 2.0 version label.</b>
```

Comment:
Thanks for the update, but I don't agree with the verbiage here. "specification is finished" gives a sense of long-term stability that we explicitly do not want to imply. Additionally, we don't want to lean to heavily on the 2.0 verbiage in public docs, as it again implies that this version is stable and will stick around in tandem to the new API.

I'd suggest something like:
"Development of this version of the WebVR API has halted, and is expected to be replaced by a newer variant (possibly link to latest spec here?) of the API. Several browser will continue to support this version of the API in the meantime."

Reply:
How about "Design of the 1.1 version of the WebVR API has halted."

Reply:
How about

`"Development of this version of the WebVR API has halted in favor of being replaced by a newer variant of the API. See latest specification draft [here](https://w3c.github.io/webvr/spec/latest/). Several browsers will continue to support this version of the API in the meantime."`

to directly link that WebVR 2.0 is the reason for WebVR 1.1 not being worked on anymore, and that there is no pause implied to occur in development.

Reply:
That sounds good to me!